src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -299,6 +301,23 @@ public sealed class HttpRequestOptions : System.Collections.Generic.IDictionary<
         public void Set<TValue>(HttpRequestOptionsKey<TValue> key, TValue value) { throw null; }
     }
 
+    [SupportedOSPlatform("windows")]
+    [SupportedOSPlatform("linux")]
+    [SupportedOSPlatform("macos")]
+    [UnsupportedOSPlatform("browser")]
+
+    public sealed class Http3WebtransportSession : System.IAsyncDisposable, IDisposable
+    {
+        public long Id { get { throw null; } }
+        internal Http3WebtransportSession() { }
+        public static System.Threading.Tasks.ValueTask<Http3WebtransportSession?> ConnectAsync(Uri uri, HttpMessageInvoker invoker, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public bool TryGetIncomingWebtransportStreamFromServer(out System.Net.Quic.QuicStream? quicStream) { throw null; }
+        public System.Threading.Tasks.ValueTask<System.Net.Quic.QuicStream?> GetIncomingWebtransportStreamFromServerAsync(CancellationToken cancellationToken = default) { throw null; }
```

Comment:
Now that you have a cancellationToken to cancel an ongoing operation, I think you don't have to have additional "TryGet.." method, but it's debatable 🤔

Also nit on namings: I believe the term usually used is "accept" instead of "get" for incoming streams. I also think, as you're within "WebtransportSession", you can skip "Webtransport" in method names -- it is obvious from class name. Same way "FromServer" part also seems obvious. So the names can be `AcceptInboundStreamAsync` and `OpenOutboundStreamAsync`, to correspond to `QuicConnection`, or just `AcceptStreamAsync` and `OpenStreamAsync`

Reply:
I solved the nit but I think that the TryGet method should still exist, maybe someone would want to see if there is a stream currently, without waiting for it